packages/sw-lib/src/lib/sw-lib.js

```diff
@@ -17,36 +17,51 @@
 
 import RouterWrapper from './router-wrapper.js';
 import ErrorFactory from './error-factory.js';
-import {PrecacheManager}
-  from '../../../sw-precaching/src/index.js';
+import {PrecacheManager} from '../../../sw-precaching/src/index.js';
+import {
+  CacheFirst, CacheOnly, NetworkFirst, NetworkOnly, StaleWhileRevalidate,
+} from '../../../sw-runtime-caching/src/index.js';
 
 /**
  * This is a high level library to help with using service worker
  * precaching and run time caching.
+ *
+ * @memberof module:sw-lib
  */
 class SWLib {
   /**
-   * Initialises the classes router wrapper.
+   * Initialises an instance of SWLib. An instance of this class is
+   * accessible when the module is imported into a service worker as
+   * `self.goog.swlib`.
    */
   constructor() {
     this._router = new RouterWrapper();
     this._precacheManager = new PrecacheManager();
+    this._strategies = {
```

Comment:
Yes and no.

This API matches that of `sw-toolbox`, that's all I was aiming for with this as it does cover a simple use case.

There are three 1 options:

1. This approach and expect developers to use the smaller modules is desired (not ideal).
2. This approach + expose the classes as well, i.e. allow `new goog.swlib.strategies.<Name of Strategy>()`. Simple use case plus allows configuration.
3. Just include the class strategies. Only con is that is requires some boilerplate for basic use.